xbmc/platform/linux/powermanagement/LogindUPowerSyscall.cpp

```diff
@@ -25,7 +25,8 @@
 
 CLogindUPowerSyscall::CLogindUPowerSyscall()
 {
-  m_delayLockFd = -1;
+  m_delayLockSleepFd = -1;
+  m_delayLockShutdownFd = -1;
```

Comment:
use default member initializers

Reply:
Should I just update all member variables to default initialization or only my added new ones?

Reply:
the ones you added is enough